# HG changeset patch
# User jwe
# Date 747887394 0
#      Mon Sep 13 02:29:54 1993 +0000
# Node ID 0a3d698c6e6094abc39a7fb43a8646091b721823
# Parent  ef73939dc2c5d7e8f4a1a617ce458e116819c4af
[project @ 1993-09-13 02:29:54 by jwe]
(is_empty): New inline function.

diff --git a/src/pt-const.h b/src/pt-const.h
--- a/src/pt-const.h
+++ b/src/pt-const.h
@@ -683,16 +683,18 @@ public:
   ComplexMatrix complex_matrix_value (void)
     { return rep->complex_matrix_value (); }
   char *string_value (void) { return rep->string_value (); }
   Range range_value (void) { return rep->range_value (); }
 
   int rows (void) { return rep->rows (); }
   int columns (void) { return rep->columns (); }
 
+  int is_empty (void) { return (rows () == 0 || columns () == 0); }
+
   tree_constant all (void) { return rep->all (); }
   tree_constant any (void) { return rep->any (); }
   tree_constant isstr (void) { return rep->isstr (); }
 
   tree_constant convert_to_str (void) { return rep->convert_to_str (); }
 
   tree_constant cumprod (void) { return rep->cumprod (); }
   tree_constant cumsum (void) { return rep->cumsum (); }
